src/query/src/datafusion/planner.rs

```diff
@@ -93,6 +93,7 @@ where
         match statement {
             Statement::Query(qb) => self.query_to_plan(qb),
             Statement::Explain(explain) => self.explain_to_plan(explain),
+            Statement::Tql(_tql) => todo!(),
```

Comment:
This should be unreachable.

I'm wondering do we really need to add a `Tql` variant in `Statement`? `Statement` is for SQL, though TQL is sit on SQL's interface. Giving `Tql` a dedicated enum or using promql's `Expr` would be clearer IMO.

Reply:
I think the `TQL` is just like `Query` variant ; it's another query statement.